website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+page_title: Enterprise to CE downgrade Process
+description: >-
+  Guide to downgrade from consul enterprise to consul ce
+---
+
+## Introduction
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version you want.
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
+available on the [HashiCorp releases page](https://releases.hashicorp.com/consul)
+
+</Tab>
+<Tab heading="Docker">
+
+Docker containers are available on [Docker Hub](https://hub.docker.com/r/hashicorp/consul)
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the downgrade to CE
+
+1. Take a snapshot:
+
+    ```
+    consul snapshot save backup.snap
+    ```
+
```

Comment:
```suggestion

   ```
   consul snapshot save backup.snap
   ```

```
Apologies - I added one too many spaces in my last suggestion, which messed up the formatting. For future reference, it's 3 spaces, not 4. This looks weird in the GH UI, but it should fix the rendering. I'll do this in all the other places I messed it up.